.gitlab-ci.yml

```diff
@@ -784,6 +784,36 @@ library:ci-fiat_crypto_ocaml:
 library:ci-flocq:
   extends: .ci-template-flambda
 
+library:ci-interval:
+  extends: .ci-template-flambda
+  stage: stage-3
+  needs:
+  - library:ci-coquelicot
+  - library:ci-flocq
+  - library:ci-mathcomp
+  - plugin:ci-bignums
+  dependencies:
+  - library:ci-coquelicot
+  - library:ci-flocq
+  - library:ci-mathcomp
+  - plugin:ci-bignums
```

Comment:
You need the transitive needs and dependencies, ie including `build:edge+flambda`

Reply:
Thanks!

BTW, some jobs depend on build:base, some on build:edge+flambda ... any reason to pick one instead of the other?

Reply:
Although considering https://github.com/coq/coq/pull/13471 I guess only `needs` is needed